html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Me</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+      <li>
+        <a href="about-me.html">About Me</a>
+      </li>
+      <li>
+        <a href="../projects/projects.html">Projects</a>
+      </li>
+      <li>
+        <a href="../contact-us/contact-us.html">Contact Us</a>
+      </li>
+    </ul>
+    <p>Hello About Me!</p>
+    <img
+      src="/html-relative-links/images/c.png"
```

Comment:
I'll let it pass, but remember we want to take the shortest distance possible, so this should be linked using `../`